test/e2e/framework/nodes_util.go

```diff
@@ -281,43 +281,6 @@ func nodePoolsGKE() ([]string, error) {
 	return strings.Fields(stdout), nil
 }
 
-// MigTemplate (GCE-only) returns the name of the MIG template that the
-// nodes of the cluster use.
-func MigTemplate() (string, error) {
```

Comment:
confirmed this function can be removed since not used at all from https://cs.k8s.io/?q=MigTemplate&i=nope&files=&repos=